### ISSUE TITLE
Unordered list of suggestions

### ISSUE THREAD

**Issue by TatriX** [liljencrantz/crush#34]
- [ ] Rename `http` to `fetch`
- [ ] Discord channel for live communication

**Comment by liljencrantz** [comment]
You can get the type of a thing with the typeof operator, e.g. `typeof 1`. It's mentioned [here](https://github.com/liljencrantz/crush/blob/master/docs/syntax.md#operators) but could be made more prominent.

**Comment by liljencrantz** [comment]
What exactly do you mean by using data frames? Do you have a specific data frame library in mind, like pandas?

**Comment by liljencrantz** [comment]
Can you explain why you consider fetch a better command name than http? `fetch method="delete" uri` seems counter-intuitive to me.

**Comment by TatriX** [comment]
> What exactly do you mean by using data frames? Do you have a specific data frame library in mind, like pandas?

Yep, mostly pandas or data.frame from R. https://github.com/nushell/rfcs/blob/6b7d41da853ece351f8b357600a3ff489f690f5c/text/0003-data-frames.md


> Can you explain why you consider fetch a better command name than http? `fetch method="delete" uri` seems counter-intuitive to me.

Mostly because `http` isn't a verb, while `fetch` is. [fetch](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch) is also a common api on the web, so `fetch(url, {method: "DELETE"}})` reads fine. I'm mostly sure that the most common use case for that tool would be to actually fetch, i.e. to download something. And then it can be used to get data using other protocols based on uri. I also think that `http` doesn't imply `http-client`.

Perhaps we should have both, where `fetch` is a simple dispatcher to a relevant other tool based on a protocol, and if you need to do something specific to a protocol, then you use that tool directly.

And finally this isn't really important, so please feel free to ignore that idea completely.

**Comment by liljencrantz** [comment]
With regards to data frames, as far as I know, there is no way to stream them, e.g. to break them up into hundreds of little pieces and have different threads work on different parts at once, which is an important feature of a shell. As such, data frames could never *replace* the types that exist in Crush today, that are mostly geared towards handling large streams of text heavy tabular data.

I don't know if adding new types that support data frames containing numerical data as well as the required commands to make operating on them be convenient and intuitive would fit well with the language as a hole or not, but it's absolutely an idea worth exploring.

**Comment by liljencrantz** [comment]
I dislike that the name fetch doesn't tell you that the "fetcher" is limited to only http. One could put the command into an http namespace and call it like `http:fetch "http://..`, instead of what we have today which is `http "http://...`. If there are other things that make sense to put into the namespace, that makes sense, but a namespace with literally only one member feels redundant.

**Comment by TatriX** [comment]
`http:serve port=8080 ./html` would be lovely

**Comment by TatriX** [comment]
Also there could be shortcuts like `http:get example.com` and `http:post example.com (body)`
